### PR TITLE
tests: Add tmt test for bootc install with LBI

### PR DESCRIPTION
**Pull request by ckyrouac**
Along with the test, this will install the LBIs on the machine running the tests prior to running the tests. Also refactors the existing LBI test into a separate test plan to make room for the install tests.

### CODE REVIEW

File: xtask/src/xtask.rs [hunk 1]
```diff
@@ -182,6 +182,13 @@ fn test_tmt(sh: &Shell) -> Result<()> {
 
     cmd!(sh, "cargo run -p tests-integration run-vm prepare-tmt").run()?;
 
+    // pull some small images that are used for LBI installation tests
+    cmd!(
+        sh,
+        "podman pull quay.io/curl/curl-base:latest quay.io/curl/curl:latest"
```
Comment (cgwalters) [inline xtask/src/xtask.rs:188]: Minor: probably worth factoring out a `const TEST_IMAGES: &[&str] = ...` at the top of the file for increased visibility?

This one actually also relates to https://gitlab.com/fedora/bootc/examples/-/merge_requests/50#note_2041291332
which hmmm, I need to finish that PR

File: tests/containerfiles/lbi/usr/share/containers/systemd/jboss-webserver-5.image [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[Image]
+Image=registry.redhat.io/jboss-webserver-5/jws5-rhel8-operator:latest
+AuthFile=/root/auth.json
```
Comment (cgwalters) [inline tests/containerfiles/lbi/usr/share/containers/systemd/jboss-webserver-5.image:3]: Is this actually used?

**Review by cgwalters — APPROVED** [review]
Looks nice to me! Thanks for working on this, I'm excited to set a better testing foundation for the project!

File: tests/containerfiles/lbi/usr/share/containers/systemd/jboss-webserver-5.image (hunk 2 above)
Comment (ckyrouac) [reply tests/containerfiles/lbi/usr/share/containers/systemd/jboss-webserver-5.image:6]: nope, added a comment to explain that it's meant to represent a quadlet image that is not logically bound.

**Comment by cgwalters** [comment]
This just needs a `cargo fmt`

**Pull request merged**